src/services/role/services/rolesService.js

```diff
@@ -0,0 +1,101 @@
+const { NotFound } = require('@feathersjs/errors');
+const sift = require('sift');
+const { RoleModel } = require('../model');
+const { preparedRoles, unique } = require('../utils/preparedRoles');
+
+// Do not proteced this route with authentication.
+const RoleServiceHooks = {
+	before: {
+		all: [],
+		find: [],
+		get: [],
+	},
+};
+
+const filterByQuery = (roles = [], query = {}) => {
+	let result = roles;
+	const q = Object.assign({}, query);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use an object spread instead of `Object.assign` eg: `{ ...foo }`.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5258109)